client/app/components/sections/Topbar/Topbar.css

```diff
@@ -54,16 +54,29 @@
     }
   }
 
+  & button {
+    border-radius: 0;
+  }
+
   & .topbarLogo {
     max-height: var(--Topbar_logoMaxHeightMobile);
-    max-width: var(--Topbar_textLogoMaxWidth);
+    max-width: 70%;
```

Comment:
To me this looks like the `max-width` is 70% of the whole top bar container, that's probably wrong?